example/improve_xml.py

```diff
@@ -0,0 +1,129 @@
+"""
+Improving non-functional properties ::
+
+    python improve_xml.py ../sample/Triangle_fast_xml
+"""
+import sys
+import random
+import argparse
+import re
+from pyggi.base import Patch, ParseError, AbstractProgram
+from pyggi.tree import XmlEngine
+from pyggi.tree import StmtReplacement, StmtInsertion, StmtDeletion, StmtMoving
+from pyggi.algorithms import LocalSearch
+
+class MyXmlEngine(XmlEngine):
```

Comment:
This is an ad-hoc engine apparently.. I think it will be better if we locate this one inside the pyggi package. By adding additional arguments to TreeProgram, we can let users choose the granularity level of modification points. (all tags, or just the statement tags).